packages/browser-wallet/src/popup/pages/Account/SendCcd/CreateTransfer.tsx

```diff
@@ -63,7 +68,7 @@ function createDefaultValues(defaultPayload: State, accountTokens?: AccountToken
             (t) => t.id === defaultPayload.tokenId
         )?.metadata;
         token = { contractIndex: defaultPayload.contractIndex, tokenId: defaultPayload.tokenId, metadata };
-        decimals = metadata?.decimals || 0;
+        decimals = getMetadataDecimals(metadata ?? {});
```

Comment:
I assume we don't want to access metadata directly anymore, yet we do that multiple times in this file?

Reply:
You're right. Should be fixed now.